Add Jankpad

username123115 opened the pull request:
[X] I HAVE RAN DRC ON MY DESIGN WITH ZERO ERRORS (Silkscreen errors are okay)
[X] I HAVE BRANDED ALL MY PARTS (PCB, CASE) WITH MY HACKPAD'S NAME
[X] I HAVE MADE SURE MY 3D PRINTED PARTS DO NOT REQUIRE SUPPORTS (Text branding overhang is okay)
[X] I HAVE MADE SURE TO INCLUDE TOLERANCES (0.25mm)
[X] MY HACKPAD'S PCB IS UNDER 100mm x 100mm, I DO NOT HAVE MORE THAN 16X SWITCHES, 2X ROTARY ENCODERS, 1X OLED, 16X LEDS.
[X] IN MY FOLDER, THERE IS A README.MD FILE THAT CONTAINS THE FOLLOWING INFORMATION:
- Full render including all the parts
- A few sentences about the inspiration and challenges
- BOM
- Photos of the Schematic, PCB, and case

qcoral reviewed (changes requested):
awesome job - couple things:

1. would you be able to remove the libraries and link them in a text file or something? they're extremely large in the repo
2. I would double check your i/o expander - not saying there's anything wrong but people have had trouble with them in the past so I would be careful

username123115 commented:
Thank you for your feedback!
1. I've removed the large libraries in the pcb directory and added links to them in the README.md
2. Thanks for the heads up, I went ahead and checked the datasheet for the MCP23017 and my schematic and I think I should be fine

username123115 commented:
Hello! Just writing to bump this pr

GaoYeGithub commented:
Nice, interesting desgin imma dm alex since I think he counts as a approval. Mr pr: https://github.com/hackclub/hackpad/pull/387
